universal-login-relayer/lib/http/routes/authorisation.ts

```diff
@@ -17,13 +22,30 @@ const getPending = (authorisationService : AuthorisationService) =>
     return responseOf({ response: result });
   };
 
-const denyRequest = (authorisationService : AuthorisationService) =>
-  async (data: {walletContractAddress: string, body: {key: string}}) => {
+const denyRequest = (authorisationService : AuthorisationService, provider: providers.Provider) =>
+  async (data: {walletContractAddress: string, body: {key: string, signature: utils.Signature}}) => {
```

Comment:
Can we make it `data: {body: AuthorisationReuqest }`?

Reply:
Sure. What about passing provider? Kinda ugly, no?

Reply:
Yup, should be `core/AuthorisationService` that handles all the work and delegated further if needed to `integration/ethereum` and `integration/sql` if needed.
Therefore route and AuthorisationService should not need provider.